Documentation/In-depth/SystemRequirements/Index.rst

```diff
@@ -219,6 +219,17 @@ Example Configuration:
        try_files $uri $uri/ /index.php$is_args$args;
    }
 
+    # TYPO3 Backend URLs
+    location /typo3/ {
+        absolute_redirect off;
+        try_files $uri /typo3/index.php$is_args$args;
+    }
+
+    location = /typo3 {
+        return 302 /typo3/;
+    }
```

Comment:
```suggestion
```

Reply:
This does not work correctly in case of a proxy is in front. Will a an alternative above....